src/cli/annotations.rs

```diff
@@ -20,7 +20,7 @@ use super::traits::Execute;
 #[derive(Args, Clone, Debug)]
 struct Stats {
     /// PDF filepath.
-    file: PathBuf,
+    file: &Path,
```

Comment:
```suggestion
    file: PathBuf,
```
I think this is breaking the CI, is it an intentional change?